### ISSUE TITLE
scripts file content within flow replaced by `!inline script_name.inline_script.bun.ts`

### ISSUE THREAD

**Issue by whollacsek** [windmill-labs/windmill-vscode#12]
1. Open a flow.yaml file
2. Show preview
3. Click on a script node in the preview
  - All .lock files get deleted
  - All script file content get replaced by `!inline script_name.inline_script.bun.ts`

**Comment by whollacsek** [comment]
Hi Ruben, thanks for the prompt reply.

I'm not sure if I understand the expected behavior fully, but in my case
what happens is all the local .lock files are deleted when I click on a
node in the preview, and the content of all the .bun.ts files are replaced
by !inline file_name.bun.ts, meaning the code is lost forever (flow.yaml
does not contain the code either). When I `wmill sync pull` I do get the
.lock files.

I downloaded the vscode extension today and I updated my windmill instance
as well to 1.330

**Comment by rubenfiszel** [comment]
Are the files (here: script_name.inline_script.bun.ts) created as well, with the content inlined ? If not, that's probably something we ought to improve

**Comment by rubenfiszel** [comment]
but just for the record, the intended usage is that you use the cli to pull from your workspace. The CLI will pull the flow folder with the correct layout and splitted files.

**Comment by whollacsek** [comment]
Yes the CLI pull is correct with the expected layout (each script with
their own .bun.ts files and .lock files) but once I click on a node in the
preview pane the folder layout is changed and the file
`script_name.inline_script.bun.ts` is modified to literally contain only
one line: `!inline script_name.inline_script.bun.ts` which means my code is
lost.

Since the content field of the flow.yaml is already `!inline
script_name.inline_script.bun.ts`, I think the script file content should
not be replaced, just kept as is.

**Comment by rubenfiszel** [comment]
I understand now. Can you check if you are on the latest version of the vscode extension ? Otherwise could you share a minimal flow that expose this behavior. I do not face it locally

**Comment by whollacsek** [comment]
I think the issue was related to missing script paths, it's something I
manually fixed then pushed so unfortunately I can not reproduce it at
the moment.
